api/logs/serializers.py

```diff
@@ -105,11 +105,13 @@ def get_params_file(self, obj):
         urls = obj.get('urls', None)
         if urls:
             view = urls.get('view', None)
-            if view:
-                file_id = view.split('/')[-2]
-                provider = view.split('/')[-3]
+            #TODO: remove the check for '//' when the urls for folder is removed from api
```

Comment:
This definitely needs a jira ticket linked to the original-- make sure you mention this TODO to remove when it is fixed.